Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -4,6 +4,11 @@ import KsApi
 import Prelude
 
 final class RewardsCollectionViewController: UICollectionViewController {
+  enum Layout {
```

Comment:
In order to decouple this and make it private for the scope of `RewardsCollectionViewController` could we move this outside of the VC body and make it private just like we do in `PledgeDescriptionCell`. Otherwise since this is internal it can be accessed from anywhere in this module like so: `RewardsCollectionViewController.Layout.Card.width`.

So the final implementation would be more consistent with what we already have.

```swift
private enum Layout {
  enum Card {
    static let width: CGFloat = 249
  }
}

final class RewardsCollectionViewController: UICollectionViewController {
  // MARK: - Properties
```